follower-read.md

```diff
@@ -34,7 +34,14 @@ This variable is used to set the expected data read mode.
 
 - When the value of `tidb_replica_read` is set to `leader` or an empty string, TiDB maintains its original behavior and sends all read operations to the leader replica to perform.
 - When the value of `tidb_replica_read` is set to `follower`, TiDB selects a follower replica of the Region to perform all read operations.
-- When the value of `tidb_replica_read` is set to `leader-and-follower`, TiDB can select any replicas to perform read operations.
+- When the value of `tidb_replica_read` is set to `leader-and-follower`, TiDB can select any replicas to perform read operations. In this mode, read requests are load balanced between the leader and follower.
+- When the value of `tidb_replica_read` is set to `closest-replicas`, TiDB prefers to select a replica in the same Region to perform read operations, which can be a leader or a follower. If there is no replica in the same Region, TiDB reads from the leader replica.
+- When the value of `tidb_replica_read` is set to `closest-adaptive`, if the estimated result of a read request is greater than or equal to the value of [`tidb_adaptive_closest_read_threshold`](/system-variables.md#tidb_adaptive_closest_read_threshold-new-in-v630), TiDB prefers to select a replica in the same Region to perform read operations. Otherwise, TiDB reads from the leader replica.
+
+> **Note:**
+>
+> - When the value of `tidb_replica_read` is set to `closest-replicas` or `closest-adaptive`, each replica of the cluster should be distributed in each Region according to the specified configuration. To configure `location-labels` for PD and set the correct `labels` for TiDB and TiKV, refer to [Schedule replicas by topology labels](/schedule-replicas-by-topology-labels.md). TiDB depends on the `zone` label to match TiKV nodes in the same Region, so you need to make sure that the `zone` label is included in the `location-labels` of PD and `zone` is included in the configuration of each TiDB and TiKV node. If your cluster is deployed using TiDB Operator, refer to [Configure replicas across Regions](https://docs.pingcap.com/tidb-in-kubernetes/stable/configure-a-tidb-cluster#high-availability-of-data).
+> - When the value of `tidb_replica_read` is set to `closest-adaptive`, to prevent the unbalanced distribution of the read traffic in various Regions, TiDB dynamically detects whether the Region distribution of all online TiDB and TiKV nodes is balanced. If a Region contains only TiDB or TiKV nodes, TiDB forces to select the leader replica to perform read operations forcefully. For example, if all TiDB nodes in a Region hang, then other online TiDB nodes are downgraded to read using leader replicas. When there is at least one TiDB node in this Region is back online, TiDB continues to prefer to select the replica in the same Region to perform read operations.
```

Comment:
```suggestion
> - When the value of `tidb_replica_read` is set to `closest-adaptive`, to prevent the unbalanced distribution of the read traffic in various Regions, TiDB dynamically detects whether the Region distribution of all online TiDB and TiKV nodes is balanced. If a Region contains only TiDB or TiKV nodes, TiDB forces to select the leader replica to perform read operations forcefully. For example, if all TiDB nodes in a Region are down, then other online TiDB nodes are downgraded to read using leader replicas. After at least one TiDB node in this Region is back online, all TiDB nodes switch back to prefer to select the replica in the same Region to perform read operations.
```